include/boost/gil/image_processing/numeric.hpp

```diff
@@ -48,6 +50,60 @@ inline double lanczos(double x, std::ptrdiff_t a)
     return 0;
 }
 
+/// \brief Generate mean kernel
+/// \ingroup ImageProcessingMath
+///
+/// Fills supplied view with normalized mean
+/// in which all entries will be equal to
+/// 1 / (dst.size())
```

Comment:
Since this is Doxygen comment, I'd suggest to mark this formula as code example using either Markdown which is supported
http://www.doxygen.nl/manual/markdown.html#md_codespan
or `\code` which I guess should also work in single line `\code ... \endcode`
http://www.doxygen.nl/manual/commands.html#cmdcode